src/main/java/com/synopsys/integration/detect/workflow/blackduck/integratedmatching/CorrelatedScanCountUploadService.java

```diff
@@ -0,0 +1,63 @@
+package com.synopsys.integration.detect.workflow.blackduck.integratedmatching;
+
+import org.apache.http.entity.ContentType;
+
+import com.google.gson.Gson;
+import com.google.gson.JsonElement;
+import com.synopsys.integration.blackduck.api.core.BlackDuckPath;
+import com.synopsys.integration.blackduck.api.core.BlackDuckResponse;
+import com.synopsys.integration.blackduck.api.generated.discovery.ApiDiscovery;
+import com.synopsys.integration.blackduck.http.BlackDuckRequestBuilder;
+import com.synopsys.integration.blackduck.service.BlackDuckApiClient;
+import com.synopsys.integration.blackduck.service.DataService;
+import com.synopsys.integration.blackduck.service.request.BlackDuckResponseRequest;
+import com.synopsys.integration.detect.workflow.blackduck.integratedmatching.model.ScanCountsPayload;
+import com.synopsys.integration.exception.IntegrationException;
+import com.synopsys.integration.log.IntLogger;
+import com.synopsys.integration.rest.HttpUrl;
+import com.synopsys.integration.rest.response.Response;
+
+public class CorrelatedScanCountUploadService extends DataService {
+    // TODO: This feature is being developed before the Black Duck endpoint is released, so this code
```

Comment:
Should we create a ticket instead of this TODO comment?

Reply:
I will leave this as a comment for now. A HUB ticket would have to be created at some point.